02 - Community Expansions/02.05 All Community Expansions/Auxiliary Tools/Open in Obsidian.md

```diff
@@ -0,0 +1,23 @@
+---
+aliases:
+  -
+tags:
+  -  evergreen
+publish: true
+---
+
+# Open in Obsidian
+
+Official website: https://routinehub.co/shortcut/11156/
+Cost: Free
+Available for:  [[MacOS Tools|MacOS]], [[iOS Apps|iOS]], [[iPadOS Apps|iPadOS]] 
+
+%% Add a description below this line. It doesn't need to be long: one or two sentences should be a good start. Mention [[🗂️ Auxiliary Tools]] or [[🗂️ 02.04 Auxiliary Tools by Category]] and any other relevat notes in this vault. %%
+
+“Open in Obsidian” is a iOS/iPadOS/MacOS [[iOS Shortcuts|Shortcut]] which takes input from the share sheet and imports it into the Obsidian app.
+
+In principle, it will accept any input. In practice, some formats will work better than others. 
```

Comment:
Is it worth letting people know how to actually use the tool? I couldn't tell from this page. From the link, it seems you have to use the QR code or something....

Reply:
I’ve updated the pull request—I think! I’m not that familiar with GitHub, and may have done something wrong.

Reply:
It worked! All good. :)